src/modals/CoreSelectModal/Styles.ts

```diff
@@ -1,12 +1,18 @@
 import styled from 'styled-components';
 
 import UModal from 'components/Modal';
+import {breakpoints} from 'styles';
 
 export const Modal = styled(UModal)`
   display: flex;
   flex-direction: column;
   max-width: 500px;
   min-width: 480px;
+
+  @media (max-width: ${breakpoints.mini}) {
+    min-width: 90%;
+    max-width: 100%;
+  }
```

Comment:
@buckyroberts Also fixed this cores modal, it was also going out on small screens.

<img width="507" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/afad8328-99d5-4e14-88de-0ee321fd4ee9">

and since this CSS might have a widespread impact as this Modal is being used at several places, so please feel free to revert back on this.
though I think this is a safe css (good for other modals too)